workflow-templates/im-deploy-on-prem-database.yml

```diff
@@ -177,6 +180,32 @@ jobs:
           DB_SERVER_NAME@stage demo: ''
           DB_SERVER_NAME@prod: ''
 
+      - name: AZ Login
+        uses: azure/login@v1
+        with:
+          # TODO: These the Service Principal these credentials are for needs access to Vault. See this git-er-done article for more information:
+          # https://github.com/im-practices/git-er-done/blob/main/actions/hashicorp-vault-integration.md#setup
+          creds: ${{ secrets.AZURE_LOGIN_CREDENTIALS }}
+
+      - name: Get Vault Token
+        id: vault_token
+        uses: im-open/get-vault-token-for-azure-user@v1.0.0
+        with:
+          # TODO: Add the role that was setup in Vault. See this git-er-done article for more information:
+          # https://github.com/im-practices/git-er-done/blob/main/actions/hashicorp-vault-integration.md#setup
+          vault-role: ''
+          vault-url: '${{ env.VAULT_URL }}'
+
+      - name: Import Secrets
+        id: vault-secrets
+        uses: hashicorp/vault-action@v2.3.1
+        with:
+          url: '${{ env.VAULT_URL }}'
+          token: '${{ steps.vault_token.outputs.client_token }}'
+          secrets: |
+            secret/data/ci/auth-creds username | SQL_AUTH_USERNAME ;
+            secret/data/ci/auth-creds password | SQL_AUTH_PASSWORD
```

Comment:
I got lots of questions on the workflow where we use `marge`.  Since this looks not right I would suggest adding a comment here so people don't mess with that formatting because they think it's a typo.

Reply:
Good point. You already asked about that in a different review. A comment would probably help cut off problems/questions before they happen.